Makefile

```diff
@@ -42,6 +42,15 @@ help:
 
 ## Build targets
 
+.PHONY: apt
+apt:
+	@echo "Installing libudunits2-dev and libnetcdf-dev ..."
+	@-bash -c "sudo apt-get install libudunits2-dev libnetcdf-dev"
+
+install-r-pkg:
+	@echo "Installing R packages ..."
+	@-bash -c './r_install.sh'
```

Comment:
I don't see `r_install.sh` should this be `install_pkgs.R` ?

Reply:
Oh, yeah. I forgot to fix Makefile. That should be `Rscript install_pkgs.R `